components/compositing/painter.rs

```diff
@@ -965,8 +957,7 @@ impl Painter {
             },
             display_list_descriptor,
         );
-        let _span =
-            profile_traits::trace_span!("ScriptToCompositorMsg::BuiltDisplayList",).entered();
+        let _span = profile_traits::trace_span!("PaintMessage::BuiltDisplayList",).entered();
```

Comment:
```suggestion
        let _span = profile_traits::trace_span!("PaintMessage::SendDisplayList",).entered();
```
`PaintMessage` doesn't have a `BuiltDisplayList` variant.